packages.json

```diff
@@ -2,6 +2,12 @@
   "packages": {
     "plugins": [
     {
+        "name": "Duplicate Line",
```

Comment:
This should match the name of the `xcodeproj`. @guillaume-algis has a PR open that will make this a bit more flexible but until that's merged it needs to be an exact match or the package won't install